web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -240,7 +240,7 @@ public static boolean startModule(Module mod, ServletContext servletContext, boo
 				if (root.getElementsByTagName("dwr").getLength() > 0) {
 					
 					// get the dwr-module.xml file that we're appending our code to
-					File f = new File(realPath + "/WEB-INF/dwr-modules.xml".replace("/", File.separator));
+					File f = new File(realPath, "WEB-INF/dwr-modules.xml".replace("/", File.separator));
```

Comment:
I think this can be replaced with `File(webInf, "dwr-modules.xml")`

check out the different constructors available https://docs.oracle.com/javase/7/docs/api/java/io/File.html
you can use a File as parent to construct a new File from it appending a child string.